src/libs/actions/Policy.ts

```diff
@@ -2590,6 +2600,336 @@ function clearCategoryErrors(policyID: string, categoryName: string) {
     });
 }
 
+function enablePolicyCategories(policyID: string, isEnabled: boolean) {
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    areCategoriesEnabled: isEnabled,
+                    pendingFields: {
+                        areCategoriesEnabled: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                    },
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    pendingFields: {
+                        areCategoriesEnabled: null,
+                    },
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    areCategoriesEnabled: !isEnabled,
+                    pendingFields: {
+                        areCategoriesEnabled: null,
+                    },
+                },
+            },
+        ],
+    };
+
+    const parameters: EnablePolicyCategoriesParams = {policyID, isEnabled};
```

Comment:
The correct API param is `enabled`. We should change it for this and all other commands
```suggestion
    const parameters: EnablePolicyCategoriesParams = {policyID, enabled};
```